index.html

```diff
@@ -0,0 +1,898 @@
+              <a href="#">The Complete 2020 Fulfillment By Amazon Mastery Bundle</a>
+            </h4>
+            <ul class="story-title">
+              <li class="color-gray"><s>$189</s>  $19</li>
+            </ul>
+          </div>
+          <div class="section-two-item">
+            <div class="section-five-item7 img-size-2 no-repeat ">
+              <div class="section-five-item-img">
+                <img src="./image/deals-ribbon.svg" alt="ribbon">
+                <div class="img-text">
+                  63% OFF
+                </div>
+              </div>
+            </div>
+            <h4 class="story-title">
+              <a href="#">Dictanote Pro: Lifetime Subscription</a>
+            </h4>
+            <ul class="story-title">
+              <li class="color-gray"><s>$346</s>  $39</li>
+            </ul>
+          </div>
+          <div class="section-two-item">
+            <div class="section-five-item8 img-size-2 no-repeat ">
+              <div class="section-five-item-img">
+                <img src="./image/deals-ribbon.svg" alt="ribbon">
+                <div class="img-text">
+                  98% OFF
+                </div>
+              </div>
+            </div>
+            <h4 class="story-title">
+              <a href="#">The Official Unity Game Development Bundle</a>
+            </h4>
+            <ul class="story-title">
+              <li class="color-gray"><s>$796</s>  $29.99</li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </section>   
+  </main>  
+  <!-- Footer section -->
+    <footer class="footer-area">
+        <div class="footer-links ">
+          <ul class="list footer-list center ">
+            <li class="list-li" ><a  href="#" class="footer-social icon" ><i class="fab fa-facebook-square "></i></a></li>
+            <li class="list-li "><a href="#" class="footer-social icon"><i class="fab fa-twitter  "></i></a></li>
+            <li class="list-li"><a href="#" class="footer-social icon"><i class="fab fa-youtube "></i></a></li>
+            <li class="list-li"><a href="#" class="footer-social icon"><i class="fab fa-instagram "></i></a></li>
+            <li class="list-li"><a href="#" class="footer-social icon"><i class="fab fa-pinterest-p "></i></a></li>
+            <li class="list-li"><a href="#" class="footer-social icon"><i class="fas fa-wifi "></i></a></li>
+            <li class="list-li"><a href="#" class="footer-social icon"><i class="fas fa-envelope "></i></a></li>
+          </ul>
+          <ul class="list footer-list center">
+            <li class="list-li"><a href="#" class="footer-list-a" >EVENTS </a></li>
+            <li class="list-li"><a href="#" class="footer-list-a">ABOUT</a></li>
+            <li class="list-li"><a href="#" class="footer-list-a">ADVERTISE</a></li>
+            <li class="list-li"><a href="#" class="footer-list-a">JOBS</a></li>
+            <li class="list-li"><a href="#" class="footer-list-a">CONTACT</a></li>
```

Comment:
- [ ] the contact icon goes to the next line on small screens
![image](https://user-images.githubusercontent.com/11758151/92158066-3fceb980-ee23-11ea-84b7-9620322fd4eb.png)